### PR TITLE
Enhance documentation on creation of a hotfix after following it!

### PR DESCRIPTION
**Pull request by jeffaco**
@MSFTOSSMgmt/ostc-devs @MSFTOSSMgmt/omsagent-devs

### CODE REVIEW

**Comment by niroyb** [comment]
Signed off

File: workflow-hotfix.md [hunk 1]
```diff
@@ -50,6 +52,53 @@ git commit -m "Fix parameters passed to omsadmin.sh for onboarding"
...
+   --or--
+git cherry-pick continue
+```
+
+- When you try to cherry pick a merged commit you may get:
+
+>
+```
+fatal: Commit e596ded is a merge but no -m option was given.
+You will want to do this to get the changes in both parents.
```
Comment (sugr4) [inline workflow-hotfix.md]: I think you can delete line 92, it's what I wrote and not from git.

**Comment by sw47** [comment]
Signed off

**Comment by sugr4** [comment]
Signed off

**Pull request merged**